examples/simple.rs

```diff
@@ -107,10 +107,15 @@ fn main() {
                 endpoint.handle(|client, _params| {
                     {
                         let cookies = client.request.cookies();
+                        #[cfg(ssl)]
```

Comment:
Sorry for the late response. Could you please elaborate why do you think that `ssl` feature is connected with cookie encryption?